src/utils/personDataUtil.ts

```diff
@@ -40,6 +40,12 @@ export const companyNamesFromPersonData = (p: PersonData): string[] => {
   return allCompaniesForPerson;
 };
 
+export const firstCompanyNameAndFnrFromPersonData = (fnr: string, p: PersonData) => {
+  return {
+    fnr, company: companyNamesFromPersonData(p).shift(),
```

Comment:
Tar du inn `fnr` her bare for å returnere det igjen?

Reply:
Til ettertanke var det unødvendig. Renamet funksjon og flyttet logikken som trengte fnr knyttet til firmanavn ut i mapping-funksjonen.